Gemfile

```diff
@@ -18,10 +18,13 @@ gem 'okcomputer'
 gem 'pg', '~> 0.18'
 gem 'sass-rails', '~> 5.0'
 gem 'site_search', '0.2.0', source: 'http://gems.dev.mas.local'
-gem 'syslog-logger'
 gem 'uglifier', '>= 1.3.0'
 gem 'unicorn-rails'
 
+group :production do
+  gem 'syslog-logger'
```

Comment:
Is there any specific reason to use the gem over `Syslog::Logger`? Asking as it seems quite an old, unmaintained repo (last commit 2012)

Reply:
Great question @MarcoBgn , didn't even occur to me this would be available in stdlib. Going to run a quick experiment on QA.

Reply:
Thanks @malcolmbaig , I'm curious to know whether the behaviour is the same

Reply:
@MarcoBgn there's some complexity here in that the unicorn config for this app also uses the gem-ified syslog. Removing it from the Gemfile causes the build to break. Adjusting the config impacts all apps, so we'd need to think about how to handle a migration away from the gem.

Given the scope of this ticket, I think it's prudent to merge these changes and perhaps revisit this issue later on.